vra/data_source_fabric_network.go

```diff
@@ -70,6 +70,10 @@ func dataSourceFabricNetwork() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"custom_properties": {
```

Comment:
Please add this property in the doc https://github.com/vmware/terraform-provider-vra/blob/master/website/docs/d/fabric_network.html.markdown